ansible/group_vars/all.yaml

```diff
@@ -9,14 +9,14 @@ lighthouse_chart_version: 0.2.5
 # for k3s traefik is installed by default
 ingress_controller: traefik
 
-# Change accordingly -> xdai or sokol
-ethereum_network: xdai
+# Change accordingly -> xdai or chiado
+ethereum_network: chiado
```

Comment:
aren't we sustituting "xdai" for "gnosis"? or not everywhere yet?

Reply:
nop, it hasnt been updated in nethermind but not in our namespaces and k8s configuration